src/js/view/utils/clickhandler.js

```diff
@@ -0,0 +1,72 @@
+define([
+    'utils/ui',
+    'events/events',
+    'utils/backbone.events',
+    'utils/underscore'
+], function(UI, events, Events, _) {
```

Comment:
Let's work out how ad click functionality should work with ads team, and how we can separate it from display UI. This is the current working implementation.